packages/mdc-tab-indicator/index.ts

```diff
@@ -81,10 +61,7 @@ class MDCTabIndicator extends MDCComponent {
     return new MDCSlidingTabIndicatorFoundation(adapter);
   }
 
-  /**
-   * @param {!ClientRect=} previousIndicatorClientRect
-   */
-  activate(previousIndicatorClientRect) {
+  activate(previousIndicatorClientRect?: ClientRect | null) {
```

Comment:
Does this have to be both `null` or `undefined` as denoted by the `?`? Can it just be `previousIndicatorClientRect?: ClientRect`?